### ISSUE TITLE
annoying Ctrl+F# instead of Alt+Ctrl+F# virtual console switching in sway with wayland

### ISSUE THREAD

**Issue by kpe** [swaywm/sway#5885]
When I press Ctrl+F# (any of the F1 to F12 keys) the display switches to the coresponding virtual console, i.e. this renders Ctrl+F# keys used in many applications unusable. Using Alt+Ctrl+F# also works. 

I cannot understand why Ctrl+F# switches to a virtual console.
I tried removing all Console bindings with something like:
```
sudo dumpkeys | grep -vE 'Console_5[^0-9]'  | sudo loadkeys
```
but it does not have an effect in sway/wayland.

I also tried `srvkeys:none`
```
setxkbmap -layout us -variant dvorak-alt-intl -option srvkeys:none
```
but it has no effect either.

It doesn't matter if I start sway from gdm or from the shell. (however in a gnome session all is fine)

Please give me a hint, how to find out where the virtual console swithing keybinding is handled! Is it in sway, or is it something else?
Is there a workaround I can try?

### Please fill out the following:
- **Sway Version:**
 - sway version 1.5.1-eca57594 (Dec 15 2020, branch 'HEAD')
 - sway version 1.5.0

- **Debug Log:**
  - config does not really matter
  - https://gist.github.com/kpe/2ff3c5f6fed8af60bf46d03afe063c4e

- **Description:**
  - The steps you took in plain English to reproduce the problem.

asked also at https://unix.stackexchange.com/questions/623873

**Comment by Emantor** [comment]
> ```
> setxkbmap -layout us -variant dvorak-alt-intl -option srvkeys:none
> ```

This does not work within sway, since we are not running within an X server. You need to configure your keyboard layout using an input directive in the sway configuration file. 
```
input "*" {
  xkb_options 'srvkeys:none'
}
```
should be worth a try. `man sway-input` has more information on the input configuration.

**Comment by kpe** [comment]
@Emantor - thanks, this solves my problem. (I was absolutely sure, that I've already tried having a `srvkeys:none` in the sway config, but it seemed to have no effect; now that it's there, all works fine):
```
input "type:keyboard" {
  xkb_layout us,de
  xkb_variant dvorak-alt-intl,
  xkb_options 'grp:alts_toggle,srvkeys:none'
}
```

**Comment by kpe** [comment]
@Emantor - there seems to be a bug after all - using `grp:alts_toggle,srvkeys:none` leads to the above annoying behaviour, I have to use `grp:shifts_toggle,srvkeys:none` to make things work as expected.

**Comment by Emantor** [comment]
> grp:alts_toggle

This indeed sounds like a bug, if it doesn't reproduce in X11.
